terraform-infra-approvals/ccd-elastic-search.json

```diff
@@ -0,0 +1,12 @@
+{
+  "resources": [],
+  "module_calls": [
+    {"source":  "git@github.com:hmcts/cnp-module-adf?ref=master"},
+    {"source":  "git@github.com:hmcts/cnp-module-waf?ref=ccd/CHG0033576"},
+    {"source":  "git@github.com:hmcts/cnp-module-storage-account.git?ref=0.0.1"},
+    {"source":  "git@github.com:hmcts/cnp-module-elk.git?ref=curator"},
+    {"source":  "git@github.com:hmcts/cnp-module-elk.git?ref=ES-7x-Upgrade"},
```

Comment:
As a convention '.git' is not used neither in the approval files nor in the tf code.